backend/api/auth.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Tuple
+
+from flask import Blueprint, jsonify, request
+from flask_jwt_extended import create_access_token
+
+from backend.database_handler_entity import mongo
+
+auth: Blueprint = Blueprint('auth', __name__)
+
+
+@auth.route('/login', methods=["POST"])
+def login() -> Tuple[Any, int]:
+    nickname = request.data.decode("utf-8")
```

Comment:
I can't understand why you use `request.data` instead of `request.get_json()` for example? If the idea that `request.data` gives us a json -- it's not correct, I just checked that. I think there is a huge mistake, you need to re-check